### PR TITLE
Finished migration from if to unless nil

### PR DESCRIPTION
**Pull request by BackSlasher**
Unless I missed something, the commit f8cd3d5af557704fd43a02c1b35251a3a31a4150 was only partial, wrecking havoc upon my servers

### CODE REVIEW

**Comment by mschile** [comment]
It seems a find and replace was done for ```if node```.  All of the ones that were missed have two spaces between ```if```  and ```node```.

**Comment by hCarmitchel** [comment]
We are experiencing the same issue described in https://github.com/chef-cookbooks/yum-mysql-community/issues/7 and this does seem to be the cause of the issue. We had to lock down to the previous version for it to work

File: recipes/connectors.rb [hunk 1]
```diff
@@ -18,26 +18,26 @@
 
 yum_repository 'mysql-connectors-community' do
   description node['yum']['mysql-connectors-community']['description'] unless node['yum']['mysql-connectors-community']['description'].nil?
-  baseurl node['yum']['mysql-connectors-community']['baseurl'] if  node['yum']['mysql-connectors-community']['baseurl'].nil?
+  baseurl node['yum']['mysql-connectors-community']['baseurl'] unless  node['yum']['mysql-connectors-community']['baseurl'].nil?
```
Comment (gravesb) [inline recipes/connectors.rb]: If you clear out the extra spaces between unless and node here and on lines 39 and 40, the find/replace that caused this issue won't miss these lines next time.

**Comment by BackSlasher** [comment]
The double spacing probably won't happen again because the rubocop will stop double spacing from happening

**Pull request merged**